# Patch for highlight
# Copyright (C) 2012 Georgi D. Sotirov <gdsotirov@dir.bg>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU General Public License
# as published by the Free Software Foundation; either version 2
# of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
# 
# You should have received a copy of the GNU General Public License
# along with this program; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
#
# Visit SlackPack at http://sotirov-bg.net/slackpack/
#
# $Id: highlight-3.9.diff,v 1.1 2012/05/05 14:03:16 gsotirov Exp $
#

# Add lua include path for the GUI
diff -urNad highlight-3.9-orig/src/gui-qt/highlight.pro highlight-3.9/src/gui-qt/highlight.pro
--- highlight-3.9-orig/src/gui-qt/highlight.pro	2011-09-01 22:37:35.000000000 +0300
+++ highlight-3.9/src/gui-qt/highlight.pro	2012-05-05 16:48:47.000000000 +0300
@@ -3,8 +3,7 @@
 # -------------------------------------------------
 TARGET = highlight-gui
 TEMPLATE = app
-INCLUDEPATH += . \
-    ../include
+INCLUDEPATH += ../include /usr/include/lua/5.1
 win32:CONFIG += static
 CONFIG += precompile_header
 PRECOMPILED_HEADER = precomp.h

# Properly detect LUA 5.1 configuration
diff -urNad highlight-3.9-orig/src/makefile highlight-3.9/src/makefile
--- highlight-3.9-orig/src/makefile	2011-10-05 23:14:22.000000000 +0300
+++ highlight-3.9/src/makefile	2012-05-05 16:48:47.000000000 +0300
@@ -33,10 +33,10 @@
 INCLUDE_DIR=./include/
 
 # Lua paths
-LUA_CFLAGS=-I/usr/include/lua5.1
-LUA_LIBS=-llua
-#LUA_CFLAGS=$(shell pkg-config --cflags lua5.1)
-#LUA_LIBS=$(shell pkg-config --libs lua5.1)
+#LUA_CFLAGS=-I/usr/include/lua5.1
+#LUA_LIBS=-llua
+LUA_CFLAGS=$(shell pkg-config --cflags lua-5.1)
+LUA_LIBS=$(shell pkg-config --libs lua-5.1)
 
 
 # Third-Party software paths
